File: hw/gudino/u2/hw19 Inheritance/Python/FarmSystem.py
# FarmSystem.py
from datetime import datetime
from Location import Location
from Cage import Cage
from Pig import Pig

def main():
    x = 15
    y = 15
    location = Location(x, y)

    cage = Cage(1, "STY FOR PIGS", 4, location)

    pig = Pig(
        4,
        "Landrace",
        datetime(2024, 9, 20),
        "male",
        True,
        120.5,
        cage
    )

    print("farmAnimal (Pig) --->", pig)

if __name__ == "__main__":
    main()

File: hw/gudino/u2/hw19 Inheritance/Js/FarmSystem.js
// FarmSystem.js
import { Location } from "./Location.js";
import { Cage } from "./Cage.js";
import { Pig } from "./Pig.js";

function main() {
    const location = new Location(15, 15);

    const cage = new Cage(1, "STY FOR PIGS", 4, location);

    const pig = new Pig(
        4,
        "Landrace",
        new Date(2024, 8, 20),
        "male",
        true,
        120.5,
        cage
    );

    console.log("farmAnimal (Pig) ---> " + pig.toString());
}

main();
